Source/SwiftLintFramework/Extensions/Configuration+Parsing.swift

```diff
@@ -45,6 +47,16 @@ extension Configuration {
             return [String].array(of: object) ?? []
         }
 
+        func defaultIndentationMode(_ object: Any?) -> IndentationMode {
```

Comment:
Alternatively this could be extracted to the IndentationMode enum as a failable initializer.